docs/monitor-emails.md

```diff
@@ -0,0 +1,56 @@
+# Monitor emails
+
+## Email types
```

Comment:
A short note about where to find these templates in the repository could be helpful, for developers?

Reply:
That’s a good point. One reason why I did not add a note that they did not have a dedicated place to point to. I think they should move into their own directory and not live between the other partials: PR #2858.